src/tpm2-provider.c

```diff
@@ -234,7 +234,6 @@ extern const OSSL_DISPATCH tpm2_file_store_functions[];
 extern const OSSL_DISPATCH tpm2_object_store_functions[];
 
 static const OSSL_ALGORITHM tpm2_stores[] = {
-    { "file", TPM2_PROPS(store), tpm2_file_store_functions },
```

Comment:
Could you please remove also the `extern const OSSL_DISPATCH tpm2_file_store_functions[]` on line 233?

Reply:
Absolutely!